src/components/leaderboard/TopThreeCard/index.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from '@/components/leaderboard/TopThreeCard/style.module.scss';
+
+interface UserCardProps {
+  position: 1 | 2 | 3;
+  state: String;
```

Comment:
```suggestion
  state: string;
```

string should be a lowercase type to represent the default string value in typescript